test/integration_tests/integrationtest_config_server_node.cpp

```diff
@@ -177,7 +161,9 @@ TEST_F(ConfigServerNodeTest, shouldAdvertiseZonesetTopic)
 
 TEST_F(ConfigServerNodeTest, shouldPublishLatchedOnZonesetTopic)
 {
-  SubscriberMock subscriber_mock;
+  ros::NodeHandle nh;
+  SubscriberMock<ros::MessageEvent<ZoneSetConfiguration const>> subscriber_mock(
+      nh, "/test_ns_laser_1/zoneconfiguration", 10);
```

Comment:
Same as above. Please consider introducing constants for topic name string and queue size. Alternatively you could consider defining `class ZoneSetSubscriberMock : public SubsciberMock<...>`. This would again hide away the whole details of the construction.

Reply:
This comment applies to all modified tests.